File: src/Einstein.WebUI/ClientApp/components/skills/rings/models/Ring.js

export class Ring {
	constructor({
		seqno = 0,

		status = {
			valid: true,
			added: false,
			modified: false,
			removed: false,
			loaded: false
		},

		id = -1,
		deleted = false,

		name = '',
		description = '',

		} = {}) {

		this.seqno = seqno;
		this.status = status;

		this.id = id;
		this.deleted = deleted;

		this.name = name;
		this.description = description;

	}
}

export function createRing(data) {
	return Object.freeze(new Ring(data));
}

File: src/Bloemert.Data.Entity.Skills/Entity/Ring.cs
using Bloemert.Data.Core;
using System;

namespace Bloemert.Data.Entity.Skills.Entity
{
	public class Ring : BaseEntity
	{
		public virtual int Seqno { get; set; }

		public virtual string Name { get; set; }

		public virtual string Description { get; set; }

	}
}
